internal/commands/commands.go

```diff
@@ -274,4 +275,20 @@ your logs and follow any newly created logs in real-time.`,
 			},
 		},
 	}
+
+	Schema = cli.CommandDefinition{
+		Use:         "schema",
+		Aliases:     []string{"schemas"},
+		Description: "Manage the schemas of your Realm app",
+		SubCommands: []cli.CommandDefinition{
+			{
+				Command:     &schema.CommandModels{},
+				Use:         "models",
+				Aliases:     []string{"model"},
+				Display:     "schema models",
+				Description: "Generate source code for the schema data models",
+				Help:        `TODO: this`,
```

Comment:
What's the plan for this TODO?

Reply:
was going to reach out to @kpham1231 for this one